### PR TITLE
Fix all warnings

### LINKED ISSUE #54: Warning pollution from gs
Let's investigate disabling warning 10369 or filtering it in python.

### PR DESCRIPTION
**Pull request by wait-wtf**
Second try.

Closes #54

### CODE REVIEW

File: source/gamespy/common/gsUdpEngine.c [hunk 1]
```diff
@@ -3,6 +3,9 @@
 // UDP Communication Engine
 #include "gsUdpEngine.h"
 
+// Ignore warning: (10178) function has no prototype
+#pragma warning off(10178)
```
Comment (riidefi) [inline source/gamespy/common/gsUdpEngine.c:7]: I think we can add this to the compiler options, like
-pragma "warning off(10178)"

Comment (wait-wtf) [reply source/gamespy/common/gsUdpEngine.c:7]: Good idea. Will quotes work here? I don't think `subprocess.run()` interpretes quotes. Might be wrong though.

Comment (wait-wtf) [reply source/gamespy/common/gsUdpEngine.c]: Fixed

**Comment by riidefi** [comment]
Looks good to me!

**Pull request merged**